config/config_apis.go

```diff
@@ -660,7 +660,7 @@ func prepareSectionFromMap(section string, mp interface{}) (cfgSec interface{},
 	case GeneralJSON:
 		cfgSec = new(GeneralJsonCfg)
 	case RPCConnsJSON:
-		cfgSec = make(RPCConnsJson)
+		cfgSec = &RPCConnsJson{}
```

Comment:
As best practice, please use new for creating a new object.

Reply:
Please also rename the config_apis.go to just apis.go as effectiveGo recommends.